### PR TITLE
Complete the hw 2.

### PR DESCRIPTION
**Pull request by AlinaZvarych**

### CODE REVIEW

File: src/main/java/hw_2/Task2.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+public class Task2 {
+    public static void main(String[] args) {
+        int x = 3;
+        int y = 4;
+
+        System.out.println(x + y);
+
+        System.out.println("" + x + y);
+
+
```
Comment (AntBelyi) [inline src/main/java/hw_2/Task2.java]: PLease remove extra lines

File: src/main/java/hw_2/Task2.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package hw_2;
+
+public class Task2 {
+    public static void main(String[] args) {
+        int x = 3;
+        int y = 4;
+
+        System.out.println(x + y);
+
+        System.out.println("" + x + y);
```
Comment (AntBelyi) [inline src/main/java/hw_2/Task2.java]: There is should be the output in one line
Screenshot_5

File: src/main/java/hw_2/Task2.java (hunk 1 above)
Comment (AlinaZvarych) [reply src/main/java/hw_2/Task2.java]: Done

File: src/main/java/hw_2/Task2.java (hunk 2 above)
Comment (AlinaZvarych) [reply src/main/java/hw_2/Task2.java]: Done

**Pull request merged**